scripts/add.tokens.erc20.mjs

```diff
@@ -0,0 +1,264 @@
+import { load } from 'cheerio';
+import dotenv from 'dotenv';
+import { ethers } from 'ethers';
+import fs from 'fs/promises';
+import fetch from 'node-fetch';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import path from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const rootPath = resolve(__dirname, '../');
+
+const envFile = process.env.NODE_ENV === 'production' ? '.env' : '.env.development';
+const envPath = resolve(rootPath, envFile);
+dotenv.config({ path: envPath });
+
+const getArgValue = (argName) => {
+	const argIndex = process.argv.indexOf(argName);
+	return argIndex > -1 && argIndex < process.argv.length - 1 ? process.argv[argIndex + 1] : null;
+};
+
+const ETHERSCAN_API_KEY = getArgValue('--etherscan-api-key') || process.env.VITE_ETHERSCAN_API_KEY;
+
+if (!ETHERSCAN_API_KEY) {
+	console.error(
+		`Missing VITE_ETHERSCAN_API_KEY. Please provide it in ${envFile} or via --etherscan-api-key argument.`
+	);
+	process.exit(1);
+}
+
+const PROD_DASHBOARD_URL = 'https://sv3dd-oaaaa-aaaar-qacoa-cai.raw.icp0.io/dashboard';
```

Comment:
should we read those canister ids from dfx.json?

Reply:
i didn't even think about that! cool tks!